src/Orckestra.Composer.C1.Core/Package/Datas/Composite.Data.Types.IFolderWhiteList_administrated.xml

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Data>
+  <Add KeyName="LayoutFiles" TildeBasedPath="~\UI.Package" />
+</Data>
```

Comment:
I got the following error, during the deploy, related to IFolderWhiteList_administrated.xml file

![image](https://user-images.githubusercontent.com/57723696/81169949-69c1bd80-8fa2-11ea-9294-68fcbb3331be.png)

Probably because of xml.xml